tests/classifier.ini

```diff
@@ -0,0 +1,80 @@
+;; small classification test
+
+[main]
+name="classification"
+tf_manager=<tf_manager>
+output="tests/tmp-test-output"
+overwrite_output_dir=True
+batch_size=16
+epochs=20
+train_dataset=<train_data>
+val_dataset=<val_data>
+trainer=<trainer>
+runners=[<runner>]
+postprocess=None
+evaluation=[("classification", <accuracy>)]
+logging_period=200
+validation_period=600
+runners_batch_size=1
+random_seed=1234
+
+[tf_manager]
+class=tf_manager.TensorFlowManager
+num_threads=4
+num_sessions=1
+
+[accuracy]
+class=evaluators.accuracy.Accuracy
+
+[train_data]
+class=dataset.load_dataset_from_files
+s_source="tests/data/train.tc.en"
+s_classification="tests/data/train.words"
+preprocessors=[("source", "source_chars", processors.helpers.preprocess_char_based)]
+lazy=True
+
+[val_data]
+class=dataset.load_dataset_from_files
+s_source="tests/data/val.tc.en"
+s_classification="tests/data/val.words"
+preprocessors=[("source", "source_chars", processors.helpers.preprocess_char_based)]
+
+[encoder_vocabulary]
+class=vocabulary.from_file
+path="tests/tmp-vocab/encoder_vocab.pkl"
+
+[encoder]
+class=encoders.sentence_encoder.SentenceEncoder
+name="sentence_encoder"
+rnn_size=7
+max_input_len=10
+embedding_size=11
+dropout_keep_prob=0.5
+attention_type=decoding_function.Attention
+data_id="source"
+vocabulary=<encoder_vocabulary>
+
+[decoder_vocabulary]
+class=vocabulary.from_wordlist
+path="tests/data/classification.vocab"
+
+[decoder]
+class=decoders.sequence_classifier.SequenceClassifier
+name="decoder"
+encoders=[<encoder>]
+dropout_keep_prob=0.5
+layers: [30,10]
```

Comment:
fakt ze nespadlo, prijde mi to divny, ale beha to spravne s rovnitkem i dvojteckou